lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1277,6 +1313,34 @@ public function isNullable($fieldName)
         return $mapping !== false && isset($mapping['nullable']) && $mapping['nullable'];
     }
 
+    /**
+     * Checks if the field is insertable.
+     *
+     * @param string $fieldName The field name.
+     *
+     * @return bool TRUE if the field is insertable, FALSE otherwise.
+     */
+    public function isInsertable($fieldName)
```

Comment:
Let's add proper types to brand-new methods.

```suggestion
    public function isInsertable(string $fieldName): bool
```